app/src/main/java/nsit/app/com/nsitapp/MainActivity.java

```diff
@@ -61,6 +61,7 @@ protected void onCreate(Bundle savedInstanceState) {
         requestWindowFeature(Window.FEATURE_INDETERMINATE_PROGRESS);
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
+        appContext=getApplicationContext();
```

Comment:
What is the need of this? I don't think you are using it anywhere.

Reply:
This is being used to get the json file using the getResources() method in class utility.